backend/models/bcf.js

```diff
@@ -408,816 +464,639 @@ function getBCFMarkup(issue, account, model, unit) {
-										const simpleAttr = simpleAttrs[simpleAttrIndex];
-										if (undefined !== issue[simpleAttr]
-											&& (undefined === matchingIssue[simpleAttr] || issue[simpleAttr] !== matchingIssue[simpleAttr])) {
-											toChange.comments.push({
-												guid: utils.generateUUID(),
-												created: currentTS,
-												action: {property: simpleAttr, from: matchingIssue[simpleAttr], to: issue[simpleAttr]},
-												owner: requester.user + "(BCF Import)"
-											});
-											toChange[simpleAttr] = issue[simpleAttr];
-										}
-									}
+	if (groupAccount && groupModel && ifc_guid) {
+		if (!group) {
+			group = {};
+		}
 
-									// Attempt to merge following attributes and sort by created desc
-									const complexAttrs = ["comments", "viewpoints"];
-									for (const complexAttrIndex in complexAttrs) {
-										const complexAttr = complexAttrs[complexAttrIndex];
-										if(!matchingIssue[complexAttr]) {
-											matchingIssue[complexAttr] = issue[complexAttr];
-											toChange[complexAttr] = matchingIssue[complexAttr];
-											continue;
-										}
-										for (let i = 0; i < issue[complexAttr].length; i++) {
-											if (-1 === matchingIssue[complexAttr].findIndex(attr =>
-												utils.uuidToString(attr.guid) === utils.uuidToString(issue[complexAttr][i].guid))) {
-												matchingIssue[complexAttr].push(issue[complexAttr][i]);
-											}
-										}
-										if (matchingIssue[complexAttr].length && matchingIssue[complexAttr][0].created) {
-											matchingIssue[complexAttr] = matchingIssue[complexAttr].sort((a, b) => {
-												return a.created > b.created;
-											});
-										}
-										toChange[complexAttr] = matchingIssue[complexAttr];
-									}
-									return Issue.updateFromBCF({account, model}, matchingIssue, toChange);
-								}).catch((error) => {
-									if (error.value === responseCodes.ISSUE_NOT_FOUND.value) {
-										issue.comments.push(bcfImportNotification);
-										issuesToCreate.push(issue);
-									} else {
-										return Promise.reject(error);
-									}
-								})
-							);
-						});
+		if (name) {
+			group.name = name;
+		}
 
-						return Promise.all(saveIssueProms).then((updatedIssues) => {
-							// We need to do this synchronously to ensure the integrity of issue numbers
-							const savedIssues = [] ;
-							const issuesCreatedProm = issuesToCreate.reduce((promise, item) => {
-								return promise.then(() => {
-									return Issue.create(account, model, item).then((issue) => {
-										savedIssues.push(issue);
-									});
-								});
-							}, Promise.resolve());
-							return issuesCreatedProm.then(() => {
-								return savedIssues.concat(updatedIssues);
-							});
-						});
+		if (color) {
+			group.color = color;
+		}
 
-					}).then(savedIssues => {
+		if (!group.objects) {
+			group.objects = {};
+		}
 
-						const notifications = [];
+		if (!group.objects[groupAccount]) {
```

Comment:
i don't think there's any point of doing this, as account is static - we're always passing the same value in.